pkg/api/convertertoadmin.go

```diff
@@ -40,6 +40,11 @@ func ConvertToAdmin(cs *OpenShiftManagedCluster) *admin.OpenShiftManagedCluster
 		PeerVnetID: cs.Properties.NetworkProfile.PeerVnetID,
 	}
 
+	oc.Properties.AuthGroupSync = &admin.AuthGroupSync{
+		Admins:  &cs.Properties.AuthGroupSync.Admins,
```

Comment:
You should be returning these only when they are non-empty strings.

Reply:
ok